mysql-test/suite/rocksdb/t/rpl_row_not_found.inc

```diff
@@ -1,12 +1,14 @@
+# Facebook MySQL has slave_exec_mode=SEMI_STRICT which allows a slave to
+# 'absorb' or ignore key not found errors. Percona Server does not.
+# This test has been modified to only test that the slave SQL thread stops on
+# error as it should when running with the default slave_exec_mode=STRICT
 --source include/have_rocksdb.inc
 --source include/master-slave.inc
```

Comment:
Heavy include files (such as master-slave.inc) should be included last in the header, after the light have_* ones